app/views/support_interface/api_tokens/index.html.erb

```diff
@@ -1,22 +1,25 @@
 <% content_for :title, 'API tokens' %>
 <h1 class='govuk-heading-xl'>API tokens</h1>
 
-<%= form_with url: support_interface_api_tokens_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+<%= form_with model: VendorApiToken.new, url: support_interface_api_tokens_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+  <%= f.govuk_collection_select :provider_id, Provider.all, :id, :name, label: { text: 'Provider' } %>
```

Comment:
Bit squashed — could this control be bigger?

Reply:
I've added the least styling possible intentionally, so we can more easily update it later. Is that okay?